web/src/marked.d.ts

```diff
@@ -0,0 +1,98 @@
+declare module "marked" {
```

Comment:
Let's add a block comment at the top explaining that this is an incomplete stub, which we include here because we only use parts of the marked API and are using a fork of the upstream library.

Reply:
yes okay... thanks!